### PR TITLE
add beta div unit testing

### PR DESCRIPTION
**Pull request by serenejiang**
simulated data for beta diversities unit testing were added

### CODE REVIEW

File: evident/effect_size.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ def effect_size(mappings, alphas, betas, output, jobs, permutations,
 
         with joblib.parallel.Parallel(n_jobs=jobs, verbose=100) as par:
             par(joblib.delayed(
-                _process_column)(bf, c, fname, method)
```
Comment (antgonza) [inline evident/effect_size.py:43]: can you delete the print statement over this line, I don't think is required, right?

**Review by antgonza — COMMENTED** [review]
looks good, only one comment, but could you also update .travis.yml to use the latest qiime2 (qiime2-2018.11)? Thanks!

File: evident/effect_size.py (hunk 1 above)
Comment (serenejiang) [reply evident/effect_size.py:43]: just updated Qiime2 and also deleted the print statement.

File: .travis.yml [hunk 2]
```diff
@@ -10,10 +10,10 @@ before_install:
   - ./miniconda.sh -b
   - export PATH=/home/travis/miniconda3/bin:$PATH
 install:
-  - wget https://data.qiime2.org/distro/core/qiime2-2018.6-py35-linux-conda.yml
-  - travis_retry conda env create -n qiime2-2018.6 -q --file qiime2-2018.6-py35-linux-conda.yml
-  - rm qiime2-2018.6-py35-linux-conda.yml
-  - source activate qiime2-2018.6
+  - wget https://data.qiime2.org/distro/core/qiime2-2018.11-py35-osx-conda.yml
```
Comment (antgonza) [inline .travis.yml]: note that you are downloading the osx version vs. linux ...

Comment (serenejiang) [reply .travis.yml]: Oops. Just correct to the linux version.

**Pull request merged**